src/app/console.py

```diff
@@ -1,58 +1,188 @@
 
         for i in range(stBarab):
+            # izberi med pozicijami, ki so še na voljo
+            x, y = random.choice(moznePozicije)
             barabe.append(
                 Baraba(
-                    x=random.randint(0, 10 * self.level),
-                    y=random.randint(0, 10 * self.level),
+                    x=x,
+                    y=y,
                     hitrost=2)
             )
+            # vsakic odstrani pozicijo barabe iz moznih pozicij
+            moznePozicije.remove((x, y))
+        # izberi pozicije za heroja in princesko
+        xh, yh = random.choice(moznePozicije)
+        moznePozicije.remove((xh, yh))
+        xp, yp = random.choice(moznePozicije)
+        moznePozicije.remove((xp, yp))
 
         self.zemlja = Zemlja(
-            sirina=10 * self.level,
-            dolzina=10 * self.level,
+            sirina=sirina,
+            visina=visina,
             hero=Hero(
-                x=random.randint(0, 10 * self.level),
-                y=random.randint(0, 10 * self.level)),
+                x=xh,
+                y=yh),
+            princeska=Princeska(
+                x=xp,
+                y=yp),
             barabe=barabe
         )
 
     def draw_game(self):
-        for y in range(self.zemlja.dolzina+1):
-            for x in range(self.zemlja.sirina+1):
+        print('draw game')
+        print(f'\033[1m\033[33m{self.level} LEVEL\033[0m')
+        print(f'\033[1m\033[33mSCORE {self.rezultat} \033[0m')
+        print('---------')
+        print(f'\033[1m\033[33mPreostali koraki {self.maxKoraki} \033[0m')
+        print('-------------------')
+        for y in range(self.zemlja.visina + 1):
+            for x in range(self.zemlja.sirina + 1):
                 flag = False
                 for b in self.zemlja.barabe:
                     if b.x == x and b.y == y:
-                        print('B ',end='')
+                        print('\033[91mB\033[0m ', end='')
                         flag = True
                         break
                 if flag:
                     continue
                 if x == self.zemlja.hero.x and y == self.zemlja.hero.y:
-                    print('H ', end='')
+                    print('\033[92mH\033[0m ', end='')
+                elif x == self.zemlja.princeska.x and y == self.zemlja.princeska.y:
+                    print('\u2665 ', end='')
                 else:
-                   print('. ', end='')
+                    print('. ', end='')
             print()
 
-
+    # premik heroja z inputi
     def input(self):
-        dx = int(input("Vnesi dx: "))
-        dy = int(input("Vnesi dy: "))
-        self.zemlja.hero.premikanje(dx=dx, dy=dy)
+        global dx
+        global dy
+        self.maxKoraki -= 1
+
+        x, y = self.zemlja.hero.x, self.zemlja.hero.y
+
+        while True:
+            uporabniskiInput = input('\033[92mVnesi dx med +2 in -2:\033[0m ')
+            if uporabniskiInput == "q":
```

Comment:
Vse hard kodirane stringe spravi v enumerate ki jih potem uporabljal v programu, nič ne sme biti hardcodirano...
Poglej na primer moj config.py na serverju... https://github.com/Programerski-klub-Ljubljana/API_programerski-klub.si/blob/660d399f94fbcaff4083209024c312f8ab87a688/app/CONST.py

To narediš zaradi tega ker je potem veliko lažje iskati po programu saj uporabiš goto definition v primeru hard kodiranih stvari pa ni šans da jih najdeš če je projekt velik.